HopsanGenerator/templates/fmu_hopsan.cpp

```diff
@@ -25,9 +25,11 @@
 #include "fmu_hopsan.h"
 #include "HopsanCore.h"
 #include "HopsanEssentials.h"
+#include "HopsanTypes.h"
 #include <string>
 #include "model.hpp"
 #include <cassert>
+#include "ComponentUtilities/num2string.hpp"
```

Comment:
Map not included, I still can.build the exported fmu but this time there is some strange template error. Have not had time to.dive into what the problem is ( this time my laptop Ubuntu 18.04, GCC 7.5) could be some problem with the hstring type

Reply:
I still think that < map >  should be explicitly included since std::map is used, so that this file becomes less dependent on what the other headers are including. 
Also the system includes should preferrably be moved last  (string and cassert), not that it matters in this case.

Reply:
Ok, I added the include. We need to rearrange the order of includes in many files I think, but that can be done later.